Update README + add SOP

chrisvanrun opened the pull request:
Part of pitch:
- github.com/DIAGNijmegen/rse-roadmap/issues/409

This PR updates the MarkDown docs of the PR, describing typical usage and how the actions are determined.

sjoerdk commented on SOP.md:
```diff
@@ -0,0 +1,91 @@
+---|---|---
+U | User Optional | X (remove tags)
+M | Mandatory | determine per tag
+C | Conditional | determine per module
```
How are conditional module choices set? Are they in manual.json somehow? I don't actually know how this works.

chrisvanrun replied:
All choices are made on the tag level, for this determinant it simply does not set anything if it is M/C.

sjoerdk commented on SOP.md:
```diff
@@ -0,0 +1,91 @@
+X/Z/U* |	X unless Z or replacement of contained instance UIDs (U) is required to maintain IOD conformance (Type 3 versus Type 2 versus Type 1 sequences containing UID references) | U | Z | X
+
+
+#### 4. Tag type.
```
Remove the period.

sjoerdk commented on SOP.md:
```diff
@@ -0,0 +1,91 @@
+Type 1 | Must be present and have a valid (non-empty) value | K
+Type 2 | Must be present, but value can be empty | Z
+Type 3 | Optional — may be present or absent | X
+
```
How is type determined? Reference SOPClassUID and module. I'm actually unclear myself about module. If a tag belongs to multiple modules, which type do you take? The most restrictive right? I mean if type according to four modules is [3,3,3,1] then the tag gets type 1 (required) right?

chrisvanrun replied:
I've updated it. In short, if there are multiple different types, the determinent does not apply and does nothing.

sjoerdk reviewed (changes requested):
Looks good. It's a complicated procedure that I think is quite hard to understand but all the elements are listed and lots of the complication cannot be further simplified.

Depending on the intended audience it might be good to have a brief overview of the guiding principles behind the SOP. For implementers of this procedure it's not needed, but for people the have to sign off on it it might be very useful. The main principle would be 'don't include anything that is not strictly required by the DICOM standard'. This is stricter than the recommended standard deidentification profile, which is process-list type (default KEEP) as opposed to the allow-list (default REMOVE) here.

I could write that overview as well. We could leave it out in this pitch. But a bit more explanation on conditional handling (see comments) would be good.

chrisvanrun commented:
Thanks for reading this Sjoerd! I've added several lines to clear up the SOP.md determinents and added a guideline section that decribes the philosopy behind the procedure.

Much improved now.
